client/activities/activity-button.tsx

```diff
@@ -169,6 +169,25 @@ export const ActivityButton = React.memo(
         },
         [disabled, hotkey, onClick],
       )
+      const getLabelJsx = useMemo(() => {
+        if (disabled || !hotkey) {
+          return label
+        }
+
+        const hotkeyString = String.fromCharCode(hotkey.keyCode).toLowerCase()
+        const labelJsx = []
+        let hasFoundHotkeyChar = false
+        for (const char of label) {
+          if (!hasFoundHotkeyChar && char.toLowerCase() === hotkeyString) {
+            labelJsx.push(<u key={char}>{char}</u>)
+            hasFoundHotkeyChar = true
```

Comment:
doesn't seem like this value is used for anything?

Reply:
The purpose for this is to stop checking for more letters to underline after it's already found one. So, for example, the Ladder activity button would have both d's underlined without this check.

Reply:
Oh sorry, didn't see it being used in the loop condition!